pkg/envoy/state.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package envoy
+
+import (
+	"fmt"
+)
+
+type State interface {
+	fmt.Stringer
+}
+
+var _ State = make(CertHash, 0)
+
+// CertHash is a State used to trigger an Envoy restart after certificates have changed.
+type CertHash []byte
```

Comment:
where/how is this used?

Reply:
Oops, didn't mean to include this file in this PR. Removed.